projects/plugins/boost/app/modules/optimizations/minify/functions-helpers.php

```diff
@@ -134,11 +88,11 @@ function jetpack_boost_page_optimize_js_exclude_list() {
 
 function jetpack_boost_page_optimize_js_exclude_list_default() {
 	// WordPress core stuff, a lot of other plugins depend on it.
-	return array( 'jquery', 'jquery-core', 'underscore', 'backbone' );
+	return Minify_JS::$default_excludes;
 }
 
 function jetpack_boost_page_optimize_css_exclude_list() {
-	$exclude_list = get_option( 'page_optimize-css-exclude' );
+	$exclude_list = get_option( 'jetpack_boost_ds_minify_css_excludes' );
```

Comment:
Use `jetpack_boost_ds_get` to get DS options

Reply:
Fixed with 8629e882a06f2fba1be131df155b0cf19a669a91.